package.json

```diff
@@ -48,6 +48,7 @@
   "dependencies": {},
   "devDependencies": {
     "async": "^2.1.2",
+    "browserify": "^13.1.1",
```

Comment:
This was a missing peer-dependency of mendel-cli. Breaks tests if you checkout a clean clone and run tests.